dataFiller/src/main/java/com/inctinctools/marathon2017/socmap/datafiller/httpClient/RestClient.java

```diff
@@ -0,0 +1,28 @@
+package com.inctinctools.marathon2017.socmap.datafiller.httpClient;
+
+import com.inctinctools.marathon2017.socmap.datafiller.api.Api;
+import com.inctinctools.marathon2017.socmap.datafiller.model.ResponseServer;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.io.UnsupportedEncodingException;
+
+public class RestClient {
```

Comment:
Don't think what we need to separate RestClient here, lets move this code to OSMSourceService